### PR TITLE
Create a file for Fuquay Varina, NC

### LINKED ISSUE #4: GitHubTeacher's recommendations for things to do/eat
## Recommendations for Fuquay Varina, NC

I think we should add some recommendations for great places to eat and things to do in Fuquay
- [x] Create a branch
- [x] Add my file
- [x] Commit my file
- [x] Push to GitHub
- [x] Open a Pull Request
- [x] Collaborate
- [ ] Merge

### PR DESCRIPTION
**Pull request by githubteacher**
This PR closes #4 by adding a fuquay varina file.

### CODE REVIEW

File: githubteacher-fuquay.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Fuquay-Varina NC
+
+### Great Places to Eat
```
Comment (githubteacher) [inline githubteacher-fuquay.md:3]: Should this be title case

File: githubteacher-fuquay.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+### Fun Things to Do
```
Comment (githubteacher) [inline githubteacher-fuquay.md:9]: Any places to go hiking?

**Review by githubteacher — COMMENTED** [review]
This is a lame review

**Review by katrinsko — COMMENTED** [review]
Testing the review feature

File: githubteacher-fuquay.md (hunk 1 above)
Comment (katrinsko) [reply githubteacher-fuquay.md:3]: I like the conciseness of this header.

File: githubteacher-fuquay.md (hunk 2 above)
Comment (katrinsko) [reply githubteacher-fuquay.md:9]: Looking forward to more contents in this

**Review by katrinsko — COMMENTED** [review]
Testing the review feature again

**Comment by githubteacher** [comment]
:+1: :shipit:

**Pull request merged**